2.6.md

```diff
@@ -3374,6 +3468,81 @@ For each bid, the `nurl` attribute can contain the win notice URL. If the bidder
 
+### 4.2.4 - Object: InterestGroupAuctionIntent <b>(PROVISIONAL)</b> <a name="objectinterestgroupauctionintent"></a>
+
+Information to signal buyer participation in a potential interest group auction for a given ad slot.
+
+<table>
+  <tr>
+    <td><strong>Attribute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td><code>impid</code></td>
+    <td>string; required <b>PROVISIONAL</b></td>
+    <td>ID of ad slot represented by the corresponding bid request Imp object.  Used to link interest group buyer information to the specific ad slot.</td>
+  </tr>
+  <tr>
+    <td><code>igb</code></td>
+    <td>array; required <b>PROVISIONAL</b></td>
+    <td>One or more InterestGroupAuctionBuyer objects (Section 4.2.5).</td>
+ </tr>
+  <tr>
+    <td><code>ext</code></td>
+    <td>object</td>
+    <td>Placeholder for bidder-specific extensions to OpenRTB.</td>
+     </td>
+     </td>
+  </tr>
+</table>
+
+
+### 4.2.5 - Object: InterestGroupAuctionBuyer <b>(PROVISIONAL)</b> <a name="objectinterestgroupauctionbuyer"></a>
+
+Information for an interest group auction buyer.
+
+<table>
+  <tr>
+    <td><strong>Attribute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td><code>origin <b>PROVISIONAL</b></code></td>
+    <td>string; required</td>
+    <td>Origin of the interest group buyer to participate in the IG auction. See https://developer.mozilla.org/en-US/docs/Glossary/Origin.</td>
+  </tr>
+  <tr>
+    <td><code>maxbid <b>PROVISIONAL</b></code></td>
+    <td>double</td>
+    <td>Maximum interest group bid price expressed in CPM currency that a bidder agrees to pay if they win an in-browser interest group auction expressed expressed in the currency denoted by the `cur` attribute. Actual winning bid in the in-browser auction that determines the amount a bidder pays for the impression may be lower than this amount. This constraint allows to reduce the risks from in-browser auction bids submitted in error or reported due to fraud and abuse.</td>
+ </tr>
+  <tr>
+    <td><code>cur <b>PROVISIONAL</b></code></td>
+    <td>string;<br>default "USD"</td>
+    <td>
+    The buyer’s currency signals, an object mapping string keys to Javascript numbers. If specified, the seller will add to its auction config `perBuyerCurrencies` attribute map, keyed by the interest group buyer origin. Indicates the currency in which interest group bids will be placed. The `maxbid` should always match the `cur` value.
```

Comment:
What are JavaScript numbers?

Reply:
This language came from the PA API explainer, but here looks like a copy/paste remnant from other attributes, e.g. priority signals.